mteb/models/sentence_transformers_models.py

```diff
@@ -165,3 +165,81 @@
     supersedes="sentence-transformers/all-mpnet-base-v1",
     adapted_from=None,
 )
+
+e5_base_v2 = ModelMeta(
+    name="intfloat/e5-base-v2",
+    languages=["eng-Latn"],
+    open_weights=True,
+    revision="1c644c92ad3ba1efdad3f1451a637716616a20e8",  # can be any
+    release_date="2023-05-19",
+    n_parameters=109_000_000,
+    memory_usage=None,
+    embed_dim=768,
+    license="apache-2.0",
+    max_tokens=512,
+    reference="https://huggingface.co/intfloat/e5-base-v2",
+    similarity_fn_name="cosine",
+    framework=["Sentence Transformers", "PyTorch"],
+    use_instructions=False,
+    supersedes="intfloat/e5-base",
```

Comment:
This might make more sense for filtering if it was `superseded_by`, like how we have it for AbsTasks?

Reply:
Here did consider the discrepancy between the two approaches. `superseded_by` is easier to filter supersedes is easier to add. We could make at property on the modelMeta to get `superseeded_by`. Fine with using superseeded_by instead (probably nicer for consistency).